README.md

```diff
@@ -12,22 +12,115 @@
 ## Pre Deploy
 You need to store user management data in some table we use this table structure:
 ```
-CREATE TABLE IF NOT EXISTS users (
-  id bigserial primary key,
-  added_at timestamp with time zone DEFAULT now(),
-  email text not null UNIQUE,
-  password_hash text not null,
-  role text not null default 'user',
-  email_token uuid not null,
-  active boolean not null default false
-);
-
-CREATE TABLE IF NOT EXISTS refetch_tokens (
-  refetch_token uuid primary key,
-  user_id integer not null,
-  added_at timestamp with time zone DEFAULT now(),
-  FOREIGN KEY (user_id) REFERENCES users (id)
-);
+-- SCHEMA: user_management
+
+-- DROP SCHEMA user_management ;
+
+CREATE SCHEMA user_management
+    AUTHORIZATION postgres;
+
+
+
+-- Table: user_management."user"
+
+-- DROP TABLE user_management."user";
+
+CREATE TABLE user_management."user"
+(
+    id uuid NOT NULL DEFAULT gen_random_uuid(),
+    username text COLLATE pg_catalog."default" NOT NULL,
+    password text COLLATE pg_catalog."default" NOT NULL,
```

Comment:
why not stick with `password_hash`? It is more accurate.

Reply:
Let's go with `password`. Seem to be the standard after looking at Wordpress and ghost schemas.